### PR TITLE
add basic scaffold

### PR DESCRIPTION
**Pull request by mmou**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		<div id="ToolbarView" class="col-md-4">
+			tools
+		</div>
+
+		<div id="PadView" class="col-md-4">
+			pad
+		</div>
+	</div>
+
+	<div id="SuggestionsView" class="row">
```
Comment (bqpd) [inline index.html:35]: There was discussion as to whether suggestions should be a row below both Toolbar and Pad, or only below Toolbar. Should be easy to change, so just making a note for now.

**Comment by bqpd** [comment]
LGTM!

**Comment by olearym** [comment]
lgtm

**Comment by bqpd** [comment]
~~Relevant~~ Adorable:

**Pull request merged**